Example/SwiftUIExample/MobilePaymentsSwiftUIExample/Screens/Home/HomeViewModel.swift

```diff
@@ -0,0 +1,106 @@
+import SquareMobilePaymentsSDK
+
+#if canImport(MockReaderUI)
+import MockReaderUI
+#endif
+
+class HomeViewModel: ObservableObject, PaymentManagerDelegate {
```

Comment:
It might be interesting to convert this to using the Observable macro instead? Make it a little more up to date. But I think it's a fine follow up too.

Reply:
agree, we should use the Observable macro

Reply:
I thought @observable was only available in ios17 onwards

Reply:
https://developer.apple.com/documentation/swiftui/migrating-from-the-observable-object-protocol-to-the-observable-macro

Reply:
Never mind, I see it's updated already!